iquip/apps/builder.py

```diff
@@ -543,8 +541,8 @@ def submit(self):
         try:
             experimentArgs = self.argumentsFromListWidget(self.builderFrame.argsListWidget)
             schedOpts = self.argumentsFromListWidget(self.builderFrame.schedOptsListWidget)
-        except ValueError:
-            logger.error("The submission is rejected because of invalid arguments.")
+        except ValueError as e:
+            logger.exception("The submission is rejected because of invalid arguments: %s", e)
```

Comment:
[`logging.exception()`](https://docs.python.org/3/library/logging.html#logging.exception) logs the exception information as well, so you don't have to explicitly write it down.
For example,
```python
except ValueError:
    logger.exception("...")
```

Reply:
Oh, I see. I will fix it.